README.md

```diff
@@ -27,19 +25,13 @@ export EB_USERNAME="[your username]"
 export EB_API_KEY="[your API key]"
 ```
 
-## Usage
-
-### Evaluating/Browsing/Deleting Systems from the Command Line
+## Command-line Usage
 
-**Evaluating Systems:** The most common usage of this client will probably be to
+The most common usage of this client will probably be to
 evaluate systems on the ExplainaBoard server. You can do that from the
-command line.
+command line using a command like this.
 
-If you are using a pre-existing dataset viewable from the
-[ExplainaBoard datasets](https://explainaboard.inspiredco.ai/datasets)
-page then you can use something like the following command:
-
-```
+```shell
```

Comment:
I have some high-level comments from UX: 
it would be nice if, when a user opens a product specification, the first thing he/she sees is a very simple (e.g., shorter), straightforward example that works (arguments have been instantiated by real examples)
With the above in mind, the following suggestions could be considered:
* for the `Command-line` usage, we could provide an instantiated example (as done in python)
* I actually feel that the `Programmatic Usage` section could be put before the `Command-line`
* Also, would these be more intuitive: `Programmatic Usage` -> `Use in Python`; `Command-line` -> `Use in Shell`
* In the future, we can further simplify the pythonic example (see comment below)